src/pages/HomePage/ContactSection.jsx

```diff
@@ -0,0 +1,148 @@
+import React, { useEffect, useState } from "react";
+import {
+  Typography,
+  Box,
+  Link,
+  IconButton,
+  Divider,
+  Container,
+  Grid,
+  useTheme,
+  Stack,
+} from "@mui/material";
+import EmailIcon from "@mui/icons-material/Email";
+import { FaInstagram } from "react-icons/fa";
```

Comment:
You need to add react icons to the package.json

Reply:
added react icons